### PR TITLE
Add _vshIoMount and vshIoUmount

### PR DESCRIPTION
**Pull request by Princess-of-Sleeping**

### CODE REVIEW

File: include/psp2/vshbridge.h [hunk 1]
```diff
@@ -17,6 +17,31 @@ int _vshSblGetSystemSwVersion(SceKernelFwInfo * data);
  
 int _vshSblAimgrGetConsoleId(char CID[32]);
  
+ /**
+ *
```
Comment (d3m3vilurr) [inline include/psp2/vshbridge.h]: can you remove extra line?

File: include/psp2/vshbridge.h [hunk 2]
```diff
@@ -17,6 +17,31 @@ int _vshSblGetSystemSwVersion(SceKernelFwInfo * data);
...
+ * @param[in] id - mount id
+ * @param[in] unk1 - Unknown, set 0
+ * @param[in] unk2 - Unknown, set 0
+ * @param[in] unk3 - work buffer
+ *
+ * @return 0 on success, < 0 on error.
+ *
```
Comment (d3m3vilurr) [inline include/psp2/vshbridge.h]: also here

File: include/psp2/vshbridge.h [hunk 3]
```diff
@@ -17,6 +17,31 @@ int _vshSblGetSystemSwVersion(SceKernelFwInfo * data);
...
+ */
+int _vshIoMount(int id, int unk1, int unk2, void* unk3);
+
+/**
+ *
```
Comment (d3m3vilurr) [inline include/psp2/vshbridge.h]: here

File: include/psp2/vshbridge.h [hunk 4]
```diff
@@ -17,6 +17,31 @@ int _vshSblGetSystemSwVersion(SceKernelFwInfo * data);
...
+ * @param[in] id - mount id
+ * @param[in] unk1 - Unknown, set 0
+ * @param[in] unk2 - Unknown, set 0
+ * @param[in] unk3 - Unknown, set 0
+ *
+ * @return 0 on success, < 0 on error.
+ *
```
Comment (d3m3vilurr) [inline include/psp2/vshbridge.h]: and here

Comment (Princess-of-Sleeping) [reply include/psp2/vshbridge.h]: int _vshSblGetSystemSwVersion(SceKernelFwInfo * data);
 
int _vshSblAimgrGetConsoleId(char CID[32]);

int _vshIoMount(int id, int unk1, int permission, void* buf);
int vshIoUmount(int id, int unk1, int unk2, int unk3); 
 
int vshIdStorageIsDirty(void);
int vshIdStorageIsFormatted(void);
int vshIdStorageIsReadOnly(void);


I deleted extra rows, is this OK?

Comment (d3m3vilurr) [reply include/psp2/vshbridge.h]: i meant 
```
/**
 *
 * blah
 ...
 * @return ...
 *
 */
```
to
```
/**
 * blah
 ...
 * @return ...
 */

```

_[FastHub-Libre](https://github.com/thermatk/FastHub-Libre/)을 사용하여 Mi 5에서 보냈습니다_

**Pull request closed without merging**